README.md

```diff
@@ -235,3 +235,40 @@ validate that the pre-scripted responses matches the real responses.
 ```
 ./.ci/verify_py_test_data.sh
 ```
+
+### Running in a Docker Container
+
+The autograder can also be run from a [docker](https://www.docker.com/) container either
+by building the image from the source code with Dockerfile or pulling it from Docker Hub.
+
+The autograder container requires two mounts. The first is `/var/run/docker.sock`,
+the socket that the Docker daemon listens on. The second is the autograder's temporary directory `/tmp/autograder-temp/`.
+Both mounts rely on a POSIX system.
+
+To build the image form source code navigate to autograder-server/ directory and run:
```

Comment:
You need to note the name (tag) of the image in this comment.
Maybe: s#image#image (with the name/tag "autograder")#